server/sandbox_run.go

```diff
@@ -254,7 +254,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 
 	securityContext := req.GetConfig().GetLinux().GetSecurityContext()
 	if securityContext == nil {
-		return nil, fmt.Errorf("no security context found")
+		logrus.Warn("no security context found in config. fuck you!")
```

Comment:
Probably don't need the swear at the end...

Reply:
fixed